### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by felixfontein**
The FAQ links result in a 404, `faq/` needs to be replaced with `faq.html` to fix that.

### CODE REVIEW

File: src/cryptography/hazmat/backends/openssl/backend.py [hunk 1]
```diff
@@ -1369,7 +1369,7 @@ def load_pem_x509_certificate(self, data):
             self._consume_errors()
             raise ValueError(
                 "Unable to load certificate. See https://cryptography.io/en/la"
-                "test/faq/#why-can-t-i-import-my-pem-file for more details."
+                "test/faq.html#why-can-t-i-import-my-pem-file for more details."
```
Comment (reaperhulk) [inline src/cryptography/hazmat/backends/openssl/backend.py:1372]: All these lines exceed the char limit now so flake8 fails. 😢

Comment (felixfontein) [reply src/cryptography/hazmat/backends/openssl/backend.py:1372]: Reminds me that I opened the failing travis result in a tab to look at what exactly was failing... Will fix in a minute!

**Comment by felixfontein** [comment]
Failed CI seems to be unrelated this time.

**Review by reaperhulk — APPROVED** [review]

**Comment by felixfontein** [comment]
@reaperhulk thanks!

**Pull request merged**